packages/amplify-prompts/src/progressbars/progressbar.ts

```diff
@@ -69,7 +69,9 @@ export class ProgressBar {
     barSize: number;
 
     constructor(options: BarOptions) {
-      this.terminal = new Terminal();
+      if (options.loneWolf) {
+        this.terminal = new Terminal();
+      }
```

Comment:
what does lonewolf mean here? Does it mean that there is no terminal such that the progress bar can be attached to?

Reply:
Discussed above in a resolved comment, but it indicates if the progressbar is part of a multibar or not. If the progress bar has `loneWolf` set to true, it's a standalone will handle the rendering. If false, the parent class will take care of rendering the various progress bars. The confusing naming and leaky abstraction will be addressed in a future refactor PR.